doc/classes/ShapeCast2D.xml

```diff
@@ -140,7 +140,7 @@
 			The shape's collision mask. Only objects in at least one collision layer enabled in the mask will be detected. See [url=$DOCS_URL/tutorials/physics/physics_introduction.html#collision-layers-and-masks]Collision layers and masks[/url] in the documentation for more information.
 		</member>
 		<member name="collision_result" type="Array" setter="" getter="get_collision_result" default="[]">
-			Returns the complete collision information from the collision sweep. The data returned is the same as in the [method PhysicsDirectSpaceState2D.get_rest_info] method.
+			Returns the complete collision information from the collision sweep. The data returned is the same as in the [method PhysicsDirectSpaceState2D.get_rest_info] method, with an additional [code]collider[/code] field for the colliding object.
```

Comment:
Should mention more details of what that is, this description doesn't really explain anything, should probably be something like "as well as the object associated with `collider_id`, using `instance_from_id`" or similar

Reply:
Looking much better, won't be able to review it properly now but might later